webpack.prod.js

```diff
@@ -1,8 +1,11 @@
 var path = require("path");
 var webpack = require("webpack");
 const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
 var version = require("./package.json").version;
 
+const ASSET_PATH = process.env.ASSET_PATH || "/";
```

Comment:
Maybe here we can add a comment explaining that this will be set to this and that in gh-pages, kinto plugin, etc.

Reply:
https://github.com/Kinto/kinto-admin/pull/2372/commits/9068653c117810b6d09f11754b11160a868064d8